templates/Entry/entries.html.twig

```diff
@@ -233,7 +260,7 @@
                 </div>
 
                 <div class="col s6">
-                    <a href="#!" class="center waves-effect waves-green btn-flat" id="clear_form_filters">{{ 'entry.filters.action.clear'|trans }}</a>
+                    <button type="reset" class="center waves-effect waves-green btn-flat">{{ 'entry.filters.action.clear'|trans }}</button>
```

Comment:
again small non critical bugfix as the JS implementation was buggy, better rely on standards